taskcluster/scripts/lib/testrail_api.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
We should add a top level Mozilla license in these 
```
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

As well some descriptors as to what this script does